app/oracle-server-test/src/test/resources/reference.conf

```diff
@@ -0,0 +1,8 @@
+bitcoin-s {
```

Comment:
So why do we need to have this stuff for the oracle app config? 

Is this just a hard requirement for anything that uses a db? 

We should look into refactoring this in the future if that is the case (file an issue).

Reply:
Oh, this was accidentially copied over from the app server test dir